notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "FYI: glupyter version: 0.2.dev143+g99ecd9e\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "863ca3ff7d3245c49336e9fd1fd48939",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Html(children=[Toolbar(children=[ToolbarItems(children=[BasicJupyterToolbar(children=[Tooltip(bottom=True, chi…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<glue_jupyter.bqplot.profile.viewer.BqplotProfileView at 0x7fd34569edf0>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# THIS CELL IS FOR ILLUSTRATION OF WHAT THE UI LOOKS LIKE FOR EXPLAINING THE CONCEPTS, NOT INTENDED AS AN ACTUAL NOTEBOOK API FOR USERS.  IT PROBABLY NEEDS TO BE EXECUTED TO ACTUALLY SHOW ANYTHING THOUGH\n",
+    "import glue_jupyter\n",
+    "print('FYI: glupyter version:', glue_jupyter.__version__)\n",
+    "from glue_jupyter import jglue\n",
+    "from astropy.io import fits\n",
+    "\n",
+    "app = jglue()\n",
+    "data = app.add_data(spectrum=spec)\n",
+    "viewer = app.profile1d(data='spectrum')\n",
+    "viewer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    There also needs to be some way to load from a file path. This should use the exact same mechanism but use a file path or URL:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.load_data(spec_url)  #<- auto-downloads from the internet\n",
+    "\n",
+    "# downalod to a local file and load it by name\n",
+    "!wget --content-disposition $spec_url\n",
+    "specviz.load_data('spec-1323-52797-0012.fits') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "    Internally this should do the same as loading the `Spectrum1D` - i.e. first convert it to `Spectrum1D` using `read` and then load it into glue/jdaviz. That way the user can know exactly what is happening when they do it with a GUI button or at the command line."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, the above should be possible using the \"open\" or \"import data\" button - the intention would be for it to all behave just as described above, but via a file browser (or command-line file path) instead of via a notebook method call."
+   ]
```

Comment:
I think anytime the user imports a spectrum, it should pop up in the viewer.   It can be unchecked afterward.  This goes towards the ongoing usability problem, related to responsiveness of the viztools and knowing that the load action has actually done something.   Alternatively, there could be a popup that says 'load successful. mark checkbox to see in viewer'

Reply:
Hmm, ok.  My thinking here was that the *first* spectrum should always appear because it's confusing to see *nothing*, but was less sure about any additional imported spectra.  So I'll update this (actually further down in the "multiple spectra in one specviz" section) to say that it will do the info popup on any imports *after the first*, as that seems to be the compromise over all of these ideas.